tests/integration/image-builders/release/Makefile

```
@@ -0,0 +1,20 @@
+.PHONY: all test build clean
+
```

checkmake (0.3.2)

[warning] Required target "test" is missing from the Makefile.

(minphony)